robert's blogpost: Key metrics for growing digital startups (#140)
xmartlabs/blog: _posts/2020-12-08-key-metrics-for-growing-digital-startups.md

---
layout: post
title: Key metrics for growing digital startups
date: 2020-12-08 10:00:00
author: Roberto Sosa
excerpt: "Which metrics should I track to refine my products and take assertive business decisions?"
tags: [digital product development, product metrics, Analytics, A/B testing, Xmartlabs]
category: development
author_id: robert
featured_image: /images/jamstack-intro/featured.jpeg
show: true
crosspost_to_medium: false
permalink: /blog/:title/
---

> Ever since the bigger tech companies like Google, Amazon and Facebook realized how powerful data can be to design impactful products that attract new users and retain older ones, they started paying more attention to which metrics they should be tracking in order to refine their products and take assertive business decisions. This resulted in the investment of these huge companies in developing analytics solutions with more reasonable pricing so smaller companies and startups could have the benefit of using data.

Reviewer: ```suggestion
Ever since the bigger tech companies like Google, Amazon and Facebook realized the powerful role data plays in designing impactful products that attracts new users and retains older ones, they started paying more attention to which metrics they should be tracking in order to refine their products and take assertive business decisions. This resulted in the investment of these huge companies in developing analytics solutions with more reasonable pricing so smaller companies and startups could have the benefit of using data.
```

> If you are in the process of growing your startup or a digital product, you may be interested in knowing how to use data the same way the biggest tech companies do to grow their own products. That is, tracking key performance indicators of two fundamental customer funnels and implementing experimental flows for different users, commonly known as A/B Testing.

Reviewer: ```suggestion
If you are in the process of growing your startup or digital product, you may be interested in getting to know how to use data the same way the biggest tech companies do to grow their own products. That is, tracking key performance indicators of two fundamental customer funnels and implementing experimental flows for different users, commonly known as A/B Testing.
```

## Understanding product evolution through funnel performance

Funnels are not a new concept in the world of sales and marketing. Funnel is a term used to describe the process of obtaining customers through different interactions with the company and it's commonly divided into different stages. Although in this article we are not going to talk about how to build a funnel, there is plenty of information out there on how to design and implement one.

> Our focus will be explaining what are the key numerical indicators you should be tracking to determine the effectiveness of the two most important funnels, the **customer acquisition funnel** and the **customer engagement funnel** as also explain how can **A/B Testing** help you improve them even further.

Reviewer: ```suggestion
Our focus will be explaining what are the key numerical indicators you should be tracking to determine the effectiveness of the two most important funnels, the **customer acquisition funnel** and the **customer engagement funnel** and also explain how can **A/B Testing** help you improve them even further.
```

### Gauging the customer acquisition funnel

> As a startup, your customer acquisition funnel is defined by your growth model, which can vary from paid models that invest in ads, to demoing or going viral in a time window so people can know your company and possibly interact with your product. Whatever model you choose to go with, they all have common metrics that can give you really good insight on how good the funnel is performing.

Reviewer: ```suggestion
As a startup, your customer acquisition funnel is defined by your growth model, which can vary from paid models that invest in ads, to demoing or going viral in a time window so people can get to know your company and possibly interact with your product. Whatever model you choose to go with, they all have common metrics that can give you really good insight on how good the funnel is performing.
```

- **Conversion Rate** **(CVR)**

    One of the most important indicators in terms of sales, it allows you to understand the relation between the number of sales and the amount of leads you are generating for your product. If your CVR is very low, chances are that something is not working as it should in your funnel and needs to be improved. Other possibility is that your business model does not depend on a high conversion rate to be profitable, in which case is not bad, but of course a higher CVR would possibly mean a higher revenue for your business.

- **Number of visits / downloads / active visitors**

    The straight forward number that tells you if both your product and funneling efforts are being effective at bringing people to buy your product/service. It's not a specific metric for all business models, but it is a good general indicator to keep an eye on to determine if you should invest more into your marketing/sales campaigns.

## Using A/B testing to improve funnels

> If you think the funneling of customers is a static process, then let me tell you that is not the case. Each acquisition and engagement strategy needs to adapt to trends and the needs of your customers, which means constantly experimenting with new features in your products. A/B Testing is the tool that can help you with this, as it offer the possibility of comparing the performance of the current funnel process (control group) with a modified process which you suspect is better (experimental group).

Reviewer 2: ```suggestion
If you think the funneling of customers is a static process, then let me tell you that is not the case. Each acquisition and engagement strategy needs to adapt to trends and the needs of your customers, which means constantly experimenting with new features in your products. A/B Testing is the tool that can help you with this, as it offers the possibility of comparing the performance of the current funnel process (control group) with a modified process which you suspect is better (experimental group).
```

> Different flows means that you can measure changes in the performance indicators of the users, for example their conversion rate. Does it change between the users in experimental flow and control one? How? Did their churn rate or abandon rate went up? why? There are common hypothetical questions that allow you to understand if the implemented features in your product are working as you theorized before you released to you whole customer base and risking the loss of customers by having a wrong understanding of your customer's needs.

Reviewer: ```suggestion
Different flows means that you can measure changes in the performance indicators of the users, for example their conversion rate. Does it change between the users in experimental flow and control one? How? Did their churn rate or abandon rate go up? Why? These are common hypothetical questions that allow you to understand if the implemented features in your product are working as you theorized before you release to your whole customer base, risking the loss of customers by having a wrong understanding of their needs.
```

> These process of understanding empirically your customers and the way they interact with your product is what nowadays is called data-driven product development. But in order to this in the most efficient way possible you will need the right analytics tool.

Reviewer 2: ```suggestion
This process of understanding empirically your customers and the way they interact with your product is what nowadays is called data-driven product development. But in order to this in the most efficient way possible you will need the right analytics tool.
```

## Implement the correct analytics tools to track your funnels and tests

> All these numbers should be tracked in the most efficient way possible. This not also means keeping an eye on the numbers, but also analyzing them and taking action. For this the data should reach out to the correct persons in daily, weekly, monthly cycles.

Reviewer: ```suggestion
All these numbers should be tracked in the most efficient way possible. This not only means keeping an eye on the numbers, but also analyzing them and taking action. For this, data should get to the correct stakeholder in daily, weekly, monthly cycles.
```

Having the best analytics platform that fits your needs and makes well-rounded dashboards that display information in a digestible and non-time-consuming way is a key technical requirement. The problem with this, is that startups maybe don't have the budget for an analytics platform when they are starting. This can be solved using the free version of current Google Analytics (Merge between GA and Firebase Analytics) or open-source analytics platforms.

> Google Analytics for Firebase can cover most needs that a startup can have when starting to develop their product. The best part is, that it works on tracking both mobile and web apps as also being very user friendly. Also, the setup is very simple in comparison to others.

Reviewer: ```suggestion
Google Analytics for Firebase can cover most startups needs when starting to develop their product. The best part is that it works for both mobile and web apps while also being very user friendly. Additionally, the setup is very simple in comparison to others.
```

The disclaimer here is that, if your startup scales maybe you should start considering better product analytics tools like Amplitude, Mixpanel, Looker, or even Tableau for interconnecting all your data sources.

For A/B Testing tools, Firebase once again has a really good product for performing such tests, the utility that Firebase for A/B Testing brings to the table is excellent as a starting tool. Another common tool you will see in this experimental phase is Optimizely, which comes with a great UI to setup your A/B experimentation and a lot of versatility outside of the technological sprectrum. Optimizely as a platform offers so many diverse solutions, from marketing ad tests to finance flows.

*Do you need help implementing the correct business intelligence tool for your business? We are [here](https://xmartlabs.com) to help!*

## The importance of monitoring all these components with analytics

> As a startup, one of your constant goals and possibly a big challenge is refining your product continuously at the same time you are growing customer base. Doing this as a product manager can be tough, and many people roll-out their products without any strategy hoping for the best. This frequently ends in a failure that could have been avoided having the right information at the right time. Maintaining good tracking of your funnels and performing A/B testing helps you realize the actual impact that both your product and startup are having and allows you to realize when and how to pivot your product based on empirical evidence of the behaviour of your customers.

Reviewer: ```suggestion
As a startup, one of your constant goals and possibly a big challenge is refining your product continuously at the same time you are growing your customer base. Doing this as a product manager can be tough, and many people roll-out their products without any strategy hoping for the best. This frequently ends in a failure that could have been avoided having the right information at the right time. Maintaining good tracking of your funnels and performing A/B testing helps you realize the actual impact that both your product and startup are having and allows you to realize when and how to pivot your product based on empirical evidence of the behaviour of your customers.
```

> The metrics mentioned in this blog are not the only ones you should be tracking, but it's usually a good place to start using analytics since most of these KPIs are almost standard between businesses. Another important thing to mention is that you should not be tracking a hundred indicators because you will inevitably lose attention of the important things. The goal of all this, is to be able to quickly analyze the big picture of the interactions with your customers and take decisions that help you mitigate problems and discover new opportunities.

Reviewer: ```suggestion
The metrics mentioned in this post are not the only ones you should be tracking, but it's usually a good starting point since most of these KPIs are -almost- standard between businesses. Another important thing to mention is that you should not be tracking a hundred indicators because you will inevitably lose attention of the important things. The goal of all this, is to be able to quickly analyze the big picture of the interactions with your customers and take decisions that help you mitigate problems and discover new opportunities.
```

> Overall, using analytics and A/B testing tools brings a lot of possibilities to the table and the cost of implementing them is now cheaper than ever, some of the tools mentioned in this blog are even free! As a digital startup, if you are not implementing any of this, you are losing a lot of potential in product refinement as algo possibly being at a disadvantage against the competition.

Reviewer: ```suggestion
Overall, using analytics and A/B testing tools brings a lot of possibilities to the table and the cost of implementing them is now cheaper than ever: some of the tools mentioned in this post are even free! As a digital startup, if you are not implementing any of this then you are potentially losing a lot of opportunities regarding product refinement as well as possibly being at a disadvantage against the competition.
```
